cfgov/locale/es/LC_MESSAGES/django.po

```diff
@@ -31,6 +31,18 @@ msgstr "Hipoteca"
 msgid "Mortgages"
 msgstr "Hipotecas"
 
+msgid "Buying a Home?"
+msgstr "¿Quiere comprar una vivienda?"
+
+msgid "Buying a House?"
+msgstr "¿Quiere comprar una vivienda?"
+
+msgid "Sign up for email tips and info to help you through the process."
+msgstr "Suscríbase a nuestra lista de email con consejos útiles sobre hipotecas."
```

Comment:
Do we need to include email heading text here, or would it be entered in the signup module?

Reply:
Not strictly needed, because you'd expect Spanish content to be entered on Spanish pages. 
I included it as a fallback because I had seen some English intros used (improperly) on Spanish pages and wanted to test the translation.